src/input/mpegts/iptv/iptv_rtsp.c

```diff
@@ -272,13 +274,23 @@ iptv_rtsp_start
     hc->hc_rtsp_user = strdup(u->user);
   if (u->pass)
     hc->hc_rtsp_pass = strdup(u->pass);
-
-  if (udp_bind_double(&rtp, &rtcp,
-                      LS_IPTV, "rtp", "rtcp",
-                      NULL, 0, NULL,
-                      128*1024, 16384, 4*1024, 4*1024) < 0) {
-    http_client_close(hc);
-    return SM_CODE_TUNING_FAILED;
+  
+  rtp_port = config.rtsp_udp_min_port;
```

Comment:
I believe this code will not fail if someone configures max_port smaller than min_port. It will still attempt to bind while it should just error out. Also a max_port of 0 should probably mean "unlimited"/unconfigured.

Reply:
Fixed! I made a specific check that will fail in this case.
I also make sure to set the starting port to 0 (aka random) if the max_port is at 0 (unconfigured).
So:
- setting a min port only will crash (min > max)
- setting a max port only will make the port random (starting port = 0)
- setting both will limit them as expected
- setting none (both at 0) will use a random port and not try anything else if the first binding fail, like before